src/test/java/org/apache/pirk/serialization/SerializationTest.java

```diff
@@ -75,6 +94,106 @@ public void testJavaSerDe() throws Exception
     Assert.assertTrue(deserializedDummyObject.equals(dummyRecord));
   }
 
+  @Test
+  public void testQuerierResponseSerializationDeserialization()
+  {
+    testQuerierResponseSerializationDeserialization(jsonSerializer);
+    testQuerierResponseSerializationDeserialization(javaSerializer);
```

Comment:
Is this line supposed to be here twice? If so, a comment as to why would be nice.